lib/ContainerManager.js

```diff
@@ -1055,10 +1056,19 @@ Server.prototype.storeBuildData = function(data, done) {
   return this.storeInDB('builds', data, done);
 };
 
-
 Server.prototype.storeInDB = function(key, data, done) {
   if (!data.id) {
-    data.id = format(this.flakeIdGen.next(), 'hex');
+    // If we do not have a data id then we need to produce one
+    // that is compatible with the ones generated by ProboCI.
+    // We will use the Crypto library to generate our groups
+    // of random hex numbers.
+    data.id = 
```

Comment:
The format that matches the regex is a uuid (v4). Instead of faking it with random data and building it yourself you could just return a uuid via 
https://www.npmjs.com/package/uuid.